data-formats/src/types.rs

```diff
@@ -1,4 +1,9 @@
-use std::{convert::TryInto, ops::Deref, str::FromStr, string::ToString};
+use std::{
+    convert::{TryFrom, TryInto},
```

Comment:
silly comment, since rust 1.56 we don't need these anymore but we're leaving them for anyone building with <1.56 right? :thinking:

Reply:
Yeah, I want to make sure things work with <1.56, since that's not yet in RHEL/Fedora.

Reply:
actually it's in fedora but yeah I see